File: Week 4/Wednesday/StretchProblemHighestNumber/StretchProblemHighestNumber/ViewController.m
//
//  ViewController.m
//  StretchProblemHighestNumber
//
//  Created by Thomas Cowern New on 12/19/18.
//  Copyright © 2018 Thomas Cowern New. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
   
    NSArray *array = @[@1, @400, @9, @3, @17, @23, @1, @8, @123, @9];
    
    NSNumber *highest = 0;
    
    for (int i = 0; i < array.count - 1; i++) {
        NSNumber *x = array[i];
        NSLog(@"%@", x);
        NSNumber *y = array[i + 1];
        NSLog(@"%@", y);
        if (x > y) {
            highest = x;
            NSLog(@"%@", highest);
        } else {
            highest = y;
        }

    }
    
    NSLog(@"%@", highest);
    
}


@end

File: Week 1/Monday/UITableViewExample/UITableViewExample/ViewController.swift
//
//  ViewController.swift
//  UITableViewExample
//
//  Created by Thomas Cowern New on 11/26/18.
//  Copyright © 2018 vetDevHouse. All rights reserved.
//

import UIKit

class ViewController: UITableViewController {

    var favoriteController = FavoriteController()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }
    
    override func numberOfSections(in tableView: UITableView) -> Int {
        return 1
    }
    
    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        return favoriteController.favorites.count
    }
    
    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
        guard let cell = tableView.dequeueReusableCell(withIdentifier: "favoritesIdentifier") else { return UITableViewCell() }
        let favorite = favoriteController.favorites[indexPath.row]
        cell.textLabel?.text = favorite
        return cell
    }
    
}

